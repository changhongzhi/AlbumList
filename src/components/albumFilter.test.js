import React from 'react';
import renderer from 'react-test-renderer';
import AlbumFilter from './albumFilter';

test('AlbumFilter change when page change', () => {
  const component = renderer.create(
    <AlbumFilter current={1} total={10} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});