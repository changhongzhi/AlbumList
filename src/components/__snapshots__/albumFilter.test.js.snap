exports[`test AlbumFilter change when page change 1`] = `
<div>
  <nav>
    <ul
      className="pagination pagination-sm col-md-4 col-md-push-4">
      <li
        className="btn-first-page  disabled">
        <a
          onClick={[Function]}>
          First
        </a>
      </li>
      <li
        className="btn-prev-page  disabled">
        <a
          onClick={[Function]}>
          «
        </a>
      </li>
      <li
        className="btn-numbered-page  active">
        <a
          onClick={[Function]}>
          1
        </a>
      </li>
      <li
        className="btn-numbered-page ">
        <a
          onClick={[Function]}>
          2
        </a>
      </li>
      <li
        className="btn-numbered-page ">
        <a
          onClick={[Function]}>
          3
        </a>
      </li>
      <li
        className="btn-numbered-page ">
        <a
          onClick={[Function]}>
          4
        </a>
      </li>
      <li
        className="btn-numbered-page ">
        <a
          onClick={[Function]}>
          5
        </a>
      </li>
      <li
        className="btn-numbered-page ">
        <a
          onClick={[Function]}>
          6
        </a>
      </li>
      <li
        className="btn-numbered-page ">
        <a
          onClick={[Function]}>
          7
        </a>
      </li>
      <li
        className="btn-next-more ">
        <a
          onClick={[Function]}>
          ...
        </a>
      </li>
      <li
        className="btn-next-page ">
        <a
          onClick={[Function]}>
          »
        </a>
      </li>
      <li
        className="btn-last-page ">
        <a
          onClick={[Function]}>
          Last
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
